Release notes 032718

JimGreene30 opened the pull request:
https://ctliotest.atlassian.net/browse/CTLIO-1118

chrislittle commented on Release Notes/2018/2018-03-27-cloud-platform-release-notes.md:
```diff
@@ -0,0 +1,45 @@
+* Load balance on any port 23 and up
+* Configurable health checks
+
+A single instance hour of the LBaaS service will be priced at $0.025. In addition, each GB of data processed by the load balancing service will be priced at $0.008.
```
we should not put pricing in kbs directly. always point to ctl.io/pricing for costs as any change in pricing requires someone to go back and find these one off figures later which is wasteful of time.

JimGreene30 commented:
I’ll fix that.

Thanks,
Jim

From: Chris Little <notifications@github.com>
Reply-To: CenturyLinkCloud/PublicKB <reply@reply.github.com>
Date: Monday, March 26, 2018 at 12:27 PM
To: CenturyLinkCloud/PublicKB <PublicKB@noreply.github.com>
Cc: "Greene, James R" <James.R.Greene@centurylink.com>, Author <author@noreply.github.com>
Subject: Re: [CenturyLinkCloud/PublicKB] Release notes 032718 (#2212)


@chrislittle commented on this pull request.

________________________________

In Release Notes/2018/2018-03-27-cloud-platform-release-notes.md<https://github.com/CenturyLinkCloud/PublicKB/pull/2212#discussion_r177154070>:

> +

+#### [Dedicated Cloud Compute (DCC)](//www.ctl.io/dedicated-cloud-compute/)

+

+##### OS Availability

+

+Due to Microsoft Support End of Life, customers can no longer Clone or Save Image for Windows Server 2003 VMs (guests) in SavvisStation Portal.

+

+#### [Load Balancer as a Service (LBaaS)](//www.ctl.io/load-balancing/)

+

+CenturyLink Cloud has released a new load balancer service in our DE1, IL1 and NY1 locations. We’re calling it Load Balancer as a Service (LBaaS). We added more features that allow you to focus on your business while we help service your infrastructure. You get to keep all of the features that are available in the current Shared Load Balancer service, and we’ve added:

+

+* TCP load balancing

+* Load balance on any port 23 and up

+* Configurable health checks

+

+A single instance hour of the LBaaS service will be priced at $0.025. In addition, each GB of data processed by the load balancing service will be priced at $0.008.

we should not put pricing in kbs directly. always point to ctl.io/pricing for costs as any change in pricing requires someone to go back and find these one off figures later which is wasteful of time.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/CenturyLinkCloud/PublicKB/pull/2212#pullrequestreview-106978239>, or mute the thread<https://github.com/notifications/unsubscribe-auth/Agu2cIcNe9TGaFAfNqQRUHMHcUPa6FMRks5tiRbogaJpZM4S7ZW->.

This communication is the property of CenturyLink and may contain confidential or privileged information. Unauthorized use of this communication is strictly prohibited and may be unlawful. If you have received this communication in error, please immediately notify the sender by reply e-mail and destroy all copies of the communication and any attachments.
